parm/config/gefs/config.cleanup

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env bash
+
+########## config.cleanup ##########
+echo "BEGIN: config.cleanup"
+
+# Get task specific resources
+source "${EXPDIR}/config.resources" cleanup
+
+export CLEANUP_COM="YES"   # NO=retain ROTDIR.  YES default in cleanup.sh
+
+#--starting and ending hours of previous cycles to be removed from rotating directory
+export RMOLDSTD=144
+export RMOLDEND=24
+
+if [[ "${DO_GEMPAK}" == "YES" ]]; then
+    export RMOLDSTD=346
+    export RMOLDEND=222
+fi
+
+# Specify the list of files to exclude from the first stage of cleanup
+# Because arrays cannot be exported, list is a single string of comma-
+# separated values. This string is split to form an array at runtime.
+case ${RUN} in
+	gdas | gfs)	exclude_string="*prepbufr*, *cnvstat*, *atmanl.nc" ;;
+	enkf*)      exclude_string="*f006.ens*" ;;
+	*)			exclude_string="" ;;
+esac
```

Comment:
This is `RUN=gefs`, so there is no `case` required here.

Reply:
Do we need a separate GEFS config file at all? Or can we just symlink the gfs version as we are doing for several others like config.com?

Reply:
that would work too.

Reply:
added the symlink